Source/santactl/Commands/SNTCommandRule.m

```diff
@@ -404,40 +408,48 @@ - (void)importJSONFile:(NSString *)jsonFilePath {
 - (void)exportJSONFile:(NSString *)jsonFilePath {
   // Get the rules from the daemon and then write them to the file.
   id<SNTDaemonControlXPC> rop = [self.daemonConn synchronousRemoteObjectProxy];
-  [rop retrieveAllRules:^(NSArray<NSDictionary *> *rules, NSError *error) {
+  [rop retrieveAllRules:^(NSArray<SNTRule *> *rules, NSError *error) {
     if (error) {
       printf("Failed to get rules: %s", [error.localizedDescription UTF8String]);
       LOGD(@"Failure reason: %@", error.localizedFailureReason);
       exit(1);
-    } else {
-      if (rules.count == 0) {
-        printf("No rules to export.\n");
-        exit(1);
-      }
-      NSError *error;
-      NSOutputStream *outputStream = [[NSOutputStream alloc] initToFileAtPath:jsonFilePath
-                                                                       append:NO];
-      [outputStream open];
-
-      // Write the rules to the file.
-      // File should look like the following JSON:
-      // {"rules": [{"policy": "ALLOWLIST", "identifier": hash, "rule_type: "BINARY"},}]}
-      NSMutableDictionary *jsonRules = [[NSMutableDictionary alloc] init];
-      jsonRules[@"rules"] = rules;
-      NSData *jsonData = [NSJSONSerialization dataWithJSONObject:jsonRules
-                                                         options:NSJSONWritingPrettyPrinted
-                                                           error:&error];
-      // Print error
-      if (error) {
-        printf("Failed to jsonify rules: %s", [error.localizedDescription UTF8String]);
-        LOGD(@"Failure reason: %@", error.localizedFailureReason);
-        exit(1);
+    }
+
+    if (rules.count == 0 || !rules) {
```

Comment:
Nit: Simplify to just `rules.count == 0`, can remove `!rules`

Reply:
Done.